autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -0,0 +1,757 @@
+        pool_group_resp = self._create_rcv3_group(lb_list=lb_pools,
+                                                  group_min=0,
+                                                  network_list=[self.rackconnect_network])
+
+        pool_group = pool_group_resp.entity
+
+        self.assertTrue(pool_group_resp.ok,
+                        msg='Create scaling group call failed with API Response: {0} for '
+                        'group {1}'.format(pool_group_resp.content, pool_group.id))
+        self.assertEquals(self.create_resp.status_code, 201,
+                          msg='The create failed with {0} for group '
+                          '{1}'.format(pool_group_resp.status_code, pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].loadBalancerId,
+                          self.pool_1.id,
+                          msg='The launchConfig for group {0} did not contain the load balancer'
+                          .format(pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].type, "RackConnectV3",
+                          msg='Load balancer type {0} is not correct for RackConnect pools'
+                          .format(pool_group.launchConfiguration.loadBalancers[0].type))
+
+    #@unittest.skip("Skipping")
+    @tags(speed='slow', type='rcv3')
+    def test_create_scaling_group_with_pool_on_private(self):
+        """
+        Test that it is possible to create a scaling group with 0 entities
+        connected to an RCV3 LB pool with only private network specified.
+        """
+        # Create a scaling group with zero servers
+        lb_pools = [{'loadBalancerId': self.pool_1.id, 'type': 'RackConnectV3'}]
+        pool_group_resp = self._create_rcv3_group(lb_list=lb_pools,
+                                                  group_min=0,
+                                                  network_list=[self.private_network])
+
+        pool_group = pool_group_resp.entity
+
+        self.assertTrue(pool_group_resp.ok,
+                        msg='Create scaling group call failed with API Response: {0} for '
+                        'group {1}'.format(pool_group_resp.content, pool_group.id))
+        self.assertEquals(self.create_resp.status_code, 201,
+                          msg='The create failed with {0} for group '
+                          '{1}'.format(pool_group_resp.status_code, pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].loadBalancerId,
+                          self.pool_1.id,
+                          msg='The launchConfig for group {0} did not contain the load balancer'
+                          .format(pool_group.id))
+        self.assertEquals(pool_group.launchConfiguration.loadBalancers[0].type, "RackConnectV3",
+                          msg='Load balancer type {0} is not correct for RackConnect pools'
+                          .format(pool_group.launchConfiguration.loadBalancers[0].type))
```

Comment:
Looks like it is; it's just cutting off before the "common scaling group assertions".